align/pnr/write_constraint.py

```diff
@@ -51,6 +51,8 @@ def map_valid_const(self,all_const):
                     const["direction"] = 'V'
                 else:
                     raise NotImplementedError(f'PnR does not support direction {const["direction"]} yet')
+            elif const["const_name"] == 'SameTemplate':
+                logger.info( f'found a SameTemplate: {const}')
```

Comment:
Should this be logger.debug?

Reply:
Eventually. We are working on a implementation of SameTemplate in the PnR engine. We can remove it when that implementation is done.